testdata/src/go.uber.org/structinit/local/local.go

```diff
@@ -28,12 +28,15 @@ type A struct {
 
 func m() {
 	var b = &A{}
+	// TODO: all errors because of "aptr" unassigned at struct initialization are grouped and reported on the line below,
+	// which is not correct. This should be fixed after https://github.com/uber-go/nilaway/issues/29 is implemented,
+	// and struct initi producer expressions are updated accordingly with the original AST expressions.
 	print(b.aptr.ptr) //want "unassigned at struct initialization"
```

Comment:
Let's add to the want regexp here a check for `"similar nil problem(s) at 6 other place(s)"`, just to make sure none of these errors go away (we can also do it for the tests above, but I am more worried about this one)

Reply:
+1

Reply:
Done